### PR TITLE
Bitmap opaque_to_polygons documentation

### PR DESCRIPTION
**Pull request by sygi**
Let's add documentation to opaque_to_polygons

Context: [link](https://godotengine.org/qa/98607/opaque_to_polygons-gives-empty-array-as-input).

### CODE REVIEW

File: doc/classes/BitMap.xml [hunk 1]
```diff
@@ -71,6 +71,12 @@
 			<argument index="1" name="epsilon" type="float" default="2.0">
 			</argument>
 			<description>
+        Creates an [code]Array[/code] of polygons covering a rectangular portion of the bitmap. Each polygon is described as a [code]PoolVector2Array[/code] of its vertices.
```
Comment (Calinou) [inline doc/classes/BitMap.xml:74]: Indentation should be done with tabs (like in the rest of the file), not spaces.

File: modules/gltf/doc_classes/GLTFTexture.xml [hunk 2]
```diff
@@ -9,7 +9,7 @@
 	<methods>
 	</methods>
 	<members>
-		<member name="src_image" type="int" setter="set_src_image" getter="get_src_image" default="212600976">
+		<member name="src_image" type="int" setter="set_src_image" getter="get_src_image" default="0">
```
Comment (Calinou) [inline modules/gltf/doc_classes/GLTFTexture.xml:12]: Don't include unrelated changes caused by doctool in the commit. (Use `git add -p` and `git reset -p` to make partial commits.)

File: doc/classes/SoftBody3D.xml [hunk 3]
```diff
@@ -99,10 +105,10 @@
 		<member name="ray_pickable" type="bool" setter="set_ray_pickable" getter="is_ray_pickable" default="true">
 			If [code]true[/code], the [SoftBody3D] will respond to [RayCast3D]s.
 		</member>
-		<member name="simulation_precision" type="int" setter="set_simulation_precision" getter="get_simulation_precision" default="0">
+		<member name="simulation_precision" type="int" setter="set_simulation_precision" getter="get_simulation_precision" default="5">
 			Increasing this value will improve the resulting simulation, but can affect performance. Use with care.
 		</member>
-		<member name="total_mass" type="float" setter="set_total_mass" getter="get_total_mass" default="0.0">
+		<member name="total_mass" type="float" setter="set_total_mass" getter="get_total_mass" default="1.0">
```
Comment (Calinou) [inline doc/classes/SoftBody3D.xml:111]: Don't include unrelated changes caused by doctool in the commit. (Use `git add -p` and `git reset -p` to make partial commits.)

The same applies to other parts modified in this file.

File: doc/classes/PhysicsServer3D.xml [hunk 4]
```diff
@@ -1543,7 +1551,9 @@
 		<constant name="SHAPE_HEIGHTMAP" value="8" enum="ShapeType">
 			The [Shape3D] is a [HeightMapShape3D].
 		</constant>
-		<constant name="SHAPE_CUSTOM" value="9" enum="ShapeType">
+		<constant name="SHAPE_SOFT_BODY" value="9" enum="ShapeType">
+		</constant>
+		<constant name="SHAPE_CUSTOM" value="10" enum="ShapeType">
```
Comment (Calinou) [inline doc/classes/PhysicsServer3D.xml:1556]: Don't include unrelated changes caused by doctool in the commit. (Use `git add -p` and `git reset -p` to make partial commits.)

The same applies to other parts modified in this file.

**Review by Calinou — CHANGES_REQUESTED** [review]

File: modules/gltf/doc_classes/GLTFTexture.xml (hunk 2 above)
Comment (sygi) [reply modules/gltf/doc_classes/GLTFTexture.xml]: Thanks for the quick turnaround. I included these changes because that's how I understood the [Updating the documentation template doc here](https://docs.godotengine.org/en/stable/community/contributing/updating_the_class_reference.html#updating-the-documentation-template):

> If there are changes to other classes than the one you are planning to document, please commit those changes first before starting to edit the template:

**Comment by sygi** [comment]
Is there anything else missing here? Should I have clicked somewhere else to re-request the review or wait more patiently? (not super familiar with github review tools)

File: doc/classes/BitMap.xml [hunk 5]
```diff
@@ -71,6 +71,12 @@
 			<argument index="1" name="epsilon" type="float" default="2.0">
 			</argument>
 			<description>
+				Creates an [code]Array[/code] of polygons covering a rectangular portion of the bitmap. Each polygon is described as a [code]PoolVector2Array[/code] of its vertices.
+				To get polygons covering the whole bitmap, pass:
+				[codeblock]
+				Rect2(0, 0, get_size().x, get_size().y)
+				[/codeblock]
+				[code]epsilon[/code] controls how accurately the polygons cover the bitmap: a lower [code]epsilon[/code] corresponds to more points in the polygons.
```
Comment (Xrayez) [inline doc/classes/BitMap.xml:79]: Maybe worth to note that this uses [marching squares](https://en.wikipedia.org/wiki/Marching_squares) as well.
```suggestion
				[code]epsilon[/code] controls how accurately the polygons cover the bitmap during marching squares algorithm: a lower [code]epsilon[/code] corresponds to more points in the polygons.
```
Or as a separate paragraph.

Comment (Xrayez) [reply doc/classes/BitMap.xml:79]: It also uses [RDP](https://en.wikipedia.org/wiki/Ramer%E2%80%93Douglas%E2%80%93Peucker_algorithm) algorithm for reducing points.

Comment (sygi) [reply doc/classes/BitMap.xml]: Not an expert here, but looking at the code and wikipedia, epsilon is a parameter of RDP, not marching squares.

**Review by Xrayez — APPROVED** [review]

**Review by Calinou — APPROVED** [review]
